en/config/actuators.md

```diff
@@ -1,139 +1,483 @@
+   - Verify that the motor is spinning very slowly in this position.
+   - If the motor is not spinning, or spinning too fast you will need to adjust the corresponding PWM `Disarmed` value in the [Actuator Outputs](#actuator-outputs) such that the motors barely spin.
+   
+     ![PWM Minimum Output](../../assets/config/actuators/pwm_minimum_output.png)
+     :::note
+     For DShot output, this is not required <!-- any, or just the minimum check? -->
+     :::
+3. Increase the slider value to a level where you can verify that the motor is spinning in the correct direction and that it would give a positive thrust in the expected direction.
+   - The expected thrust direction can vary by vehicle type, for example in multicopters the thrust should always point upwards, while in a fixed wing vehicle the thrust will push the vehicle forwards.
+   - For VTOL, thrust should point upwards when the Tilt Servo is at 0 degrees as defined the [Tilt Servo Convention](#motor-tilt-servos-conceptual).
+     Testing of the [Tilt Servo](#tilt-servo-testing) is covered below as well.
+   - If thrust is in the wrong direction, you may need to [reverse the motors](#reversing-motors).
+   
+
+### Control Surface Setup
+
+The control surface configuaration sets output values such that the surfaces:
+
+1. have the correct idle position when disarmed.
+1. move in the directions as defined in the [Control Surface Convention](#control-surface-direction-conceptual).
+
+![Control Surface Disarmed 1500 Setting](../../assets/config/actuators/control_surface_disarmed_1500.png)
+
+For each control surface:
+1. Set the `Disarmed` value for Control Surface outputs to `1500` (note, this assumes we're using a PWM actuator where 1500 is the middle of the range)
+   - Unlike motors, when `disarmed` we usually control surfaces to be in the middle position of their range.
+   - Adjust the value until the control is positioned correctly.
+2. Set the Output mode to `PWM 50Hz`, instead of default PWM 400 Hz when using a PWM output for Control Surfaces
+   - If your servo is creating weird noise, it is most likely because of the "PWM 400Hz" output, which is usually not accepted by commercial Servos.
+   - In case your Servo supports 100, 200 or 400 Hz command, you can of course set the output mode to the maximum PWM frequency it supports!
+3. Move Control Surface slider upwards (positive command) and verify that it moves in the direction defined in the [Control Surface Convention](#control-surface-direction-conceptual).
+   1. If the control surface moves in the opposite direction, click on the 'Rev Range' checkbox to reverse the range
+4. Move the slider to the middle and check if the Control Surfaces are aligned in the neutral position of the wing
+
+    ![Control Surface Trimming](../../assets/config/actuators/control_surface_trim.png)
+   1. If it is not aligned, you can set the **Trim** value for each control surface.
```

Comment:
Now we're talking about trimming. I'm even more confused

Reply:
Not sure why you are confused. This is a procedure of setting up a control surface that any user would go through :thinking: